DA-626: update AWS Bedrock

teetangh opened the pull request.

nithishr commented:
Looks good for the most part. 
Just a couple of minor things:
- You will not be able to create the bucket on Capella. Please add a note on it.
- Can you remove the INFO logs while generating the RAG response. It is quite noisy.

teetangh commented:
@nithishr 
- added note for no bucket creation on capella 
- removed the INFO logs while generating the RAG response. 
- removed the retry logs from output
- added conclusion similar to the other tutorials.
